package.json

```diff
@@ -8,8 +8,10 @@
     "@rails/webpacker": "^5.4.3",
     "glob-parent": "^6.0.2",
     "govuk-frontend": "^4.0.0",
+    "is-svg": "^4.3.0",
     "jquery": "^3.6.0",
-    "normalize-url": "7.0.2"
+    "normalize-url": "7.0.2",
+    "postcss-svgo": "4.0.3"
```

Comment:
I'd suggest we don't want the dependencies added. They're not direct dependencies and adding them here makes them look like they are. That's what the resolutions are for - to resolve indirect dependencies.